docker/start_celery_docker.sh

```diff
@@ -3,7 +3,13 @@
 cd /seed
 
 echo "Waiting for postgres to start"
-/usr/local/wait-for-it.sh --strict -t 0 db-postgres:5432
+if [ -v POSTGRES_HOST ];
+then
+   POSTGRES_ACTUAL_HOST=$POSTGRES_HOST
+else
+   POSTGRES_ACTUAL_HOST=db-postgres
+fi
+/usr/local/wait-for-it.sh --strict -t 0 $POSTGRES_ACTUAL_HOST:$POSTGRES_PORT
```

Comment:
I think this is fine for the bash portion. I'm sure there is some crazy bash command we can use, but it will definitely reduce the readability. So I like what you have!